python/lsst/atmospec/processStar.py

```diff
@@ -215,6 +245,8 @@ def setDefaults(self):
 
         self.charImage.doApCorr = False
         self.charImage.doMeasurePsf = False
+        self.charImage.repair.cosmicray.nCrPixelMax = 100000  # XXX this is new!
```

Comment:
Why these comments?

Reply:
Just because they were changes that were going into the Gen2 code while doing the Gen3 port, and I wanted to note to myself things that I'd changed there, as it was nominally a Gen2 no-op on this ticket. I've removed them now.